app/assets/javascripts/feed_entries.js

```diff
@@ -0,0 +1,13 @@
+$(document).ready(function() {
+  var fetchRole = "a[data-role='fetch']";
+  $(document).on("ajax:success", fetchRole, function (_e, _data, _status, xhr) {
+    console.log("Fetching feed");
+    $("div#flash").replaceWith(xhr.responseText);
+  })
+  .on("ajax:error", fetchRole, function(_e, _xhr, _status, _error) {
+    console.log("Error");
+  })
+  .on("ajax:complete", fetchRole, function(_e, _xhr, _status) {
```

Comment:
'_status' is defined but never used.<br>'_xhr' is defined but never used.<br>'_e' is defined but never used.